test/controllers/incorrect-information/wrong.sic.controller.unit.ts

```diff
@@ -0,0 +1,29 @@
+import mocks from "../../mocks/all.middleware.mock";
+import request from "supertest";
+import app from "../../../src/app";
+import { WRONG_SIC_PATH } from "../../../src/types/page.urls";
+
+const STOP_PAGE_TEXT = "Currently, changes to the company SIC codes can only be made by filing a confirmation statement";
+const SIC_CODE = "123";
+const SIC_CODE_DESCRIPTIONS = "Test SIC code descriptions";
+const SIC_CODE_DETAILS = "<strong>" + SIC_CODE + "</strong> - " + SIC_CODE_DESCRIPTIONS;
+
+describe("Wrong sic stop controller tests", () => {
+
+  beforeEach(() => {
+    mocks.mockAuthenticationMiddleware.mockClear();
+    mocks.mockServiceAvailabilityMiddleware.mockClear();
+    mocks.mockSessionMiddleware.mockClear();
+  });
+
+  describe("test for the get function", () => {
+
+    it("Should render the wrong sic stop page", async () => {
+      const response = await request(app).get(WRONG_SIC_PATH);
+
+      expect(response.text).toContain(STOP_PAGE_TEXT);
+      expect(response.text).not.toContain(SIC_CODE_DETAILS);
+      expect(response.text).toContain("Incorrect SIC");
+    });
+  });
```

Comment:
I'd add a test to check the back link url is correct as well

Reply:
Not sure whether there's any easy way to do this and make a test meaningful, do we usuually check for backlinks?

Reply:
i think we've done it in a couple of the tests. I'll see if I can find a way.

Reply:
I have added this as long as we just need to prove that it links back to the sic page then it should be fine.